nvflare/dashboard/application/__init__.py

```diff
@@ -23,6 +23,7 @@
 
 
 def init_app():
+    os.makedirs("/var/tmp/nvflare/dashboard", exist_ok=True)
```

Comment:
For this temp folder, why we put in "/var/tmp" but not "/tmp"?
What is the pros and cons?

If so, should we also change other "/tmp/nvflare" paths to "/var/tmp/nvflare"?

Reply:
/var/tmp folder is not removed after reboot while /tmp is.  Because the db file should not be erased after reboot, it's better inside /var/tmp/.

Reply:
got it, thanks for the explanation